### ISSUE TITLE
Onboarding: Murna Isaac

### ISSUE THREAD

**Issue by isaacmurna** [hackforla/guides#111]
### Prerequisite
- You have done the steps in this wiki page for onboarding new team members https://github.com/hackforla/guides/wiki/Joining-the-Team

### Overview
We need an unbiased review of the gather presentation so that we can improve it.

#### Additional Details
You will be working on guide during this issue, but the purpose of this issue is to evaluate and improve the Gather Examples and Interview Teams presentation.  
- Any notes you have about the guide you will be making should be noted in the issue related to the Guide you choose.
- Any notes about the Gather Examples and Interview Teams presentation should be in the document you create.

### Action Items
- [ ] Create a Google doc, with the name ```GT: Notes on Gather presentation - YOUR NAME HERE``` in the Gather Examples folder (customize with your name)
   - [ ] Add the link to the document and its name in the Resources section below
- [ ] Add to the notes document as you go, with observations about the Gather Presentation.  You will encounter some issues, such as font size or other types of formatting changes, what we are looking for though is material changes to the instructions.  For instance, if something could use a screenshot to make it clearer, and it doesn't have one or when you try to follow the steps, something is missing.
   - [ ] If while looking at the presentation (not the guide), you encounter any words that you think should be added to our glossary,  please add them to the Glossary issue as comments.
- [ ] Review the Presentation for Gathering Examples
- [ ] Pick a guide 
   - [ ] Note the Name of guide and URL of the guide in the comments below
- [ ] Use the presentation to work on the guide
   - [ ] Leaving comments in the presentation if you have a suggested improvement
   - [ ] Putting any notes about the guide either in the guide or in the issue for the guide (as appropriate)
- [ ] Clean up your presentation notes
- [ ] Present to the Guides Team

### Resources/Instructions
- #88 
- [Guides Tracker](https://github.com/orgs/hackforla/projects/3)
- [Gather Examples and Interview Teams presentation](https://docs.google.com/presentation/d/1gkPpgnOPv7lBckoLic3HAA7z_IkeTnCw9ImQp9hkujM/edit)
- [PM Templates and Guides Project board](https://github.com/hackforla/product-management/projects/2)
- Folder for Team notes and Guide Presentation: [Gather Examples](https://drive.google.com/drive/u/0/folders/15ybydhWpIYr9a3MyE2FZf1g-SJs-23Jl)
- Your presentation notes document file: 
- Guide you are working on
   - Name of guide
   - URL of guide Issue

### When you do your update please provide
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

**Comment by isaacmurna** [comment]
Progress: Little progress just because i am not sure what to do?"
Blockers: Non."
Availability: I cant say because i dont know what is expected of me.?"

**Comment by SheIsBukki** [comment]
@isaacmurna Can you mention what specifically is it or a list of items that you need clarifications for?

**Comment by isaacmurna** [comment]
Progress: i need time to read through the guide in order to give my honest review"
Blockers: Non."
Availability: 24 hours?"

**Comment by SheIsBukki** [comment]
@isaacmurna Please provide an update

Progress: "What is the current status of your project? What have you completed and what is left to do?"
Blockers: "Difficulties or errors encountered."
Availability: "How much time will you have this week to work on this issue?"
ETA: "When do you expect this issue to be completed?"
Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

**Comment by isaacmurna** [comment]
Progress: I have read the document and written down my honest review on the google docs
Blockers: Non."
Availability:

**Comment by isaacmurna** [comment]
Progress: did you get the chance to look at my review?
Blockers: "What do i do next?."
Availability: "6hrs

**Comment by SheIsBukki** [comment]
Yes, I did, we are working on breaking down the slide deck into 5 smaller sections so that it won't overwhelm users. Take a look at the 5 slide decks in this issue https://github.com/hackforla/guides/issues/112 and share your review about it

**Comment by edwardsarah** [comment]
Closing due to inactivity. If you would like to rejoin the team in the future, please reach out on slack.
